affine/__init__.py

```diff
@@ -421,41 +458,65 @@ def is_orthonormal(self) -> bool:
 
     @cached_property
     def is_degenerate(self) -> bool:
-        """True if this transform is degenerate.
+        """Return True if this transform is degenerate.
+
+        A degenerate transform will collapse a shape to an effective area
+        of zero, and cannot be inverted.
 
-        Which means that it will collapse a shape to an effective area
-        of zero. Degenerate transforms cannot be inverted.
+        Returns
+        -------
+        bool
         """
         return self.determinant == 0.0
 
     @cached_property
     def is_proper(self) -> bool:
-        """True if this transform is proper.
+        """Return True if this transform is proper.
 
-        Which means that it does not include reflection.
+        A proper transform (with a positive determinant) does not include
+        reflection.
+
+        Returns
+        -------
+        bool
         """
         return self.determinant > 0.0
 
     @property
     def column_vectors(self):
-        """The values of the transform as three 2D column vectors"""
+        """The values of the transform as three 2D column vectors.
+
+        Returns
+        -------
+        tuple of three tuple pairs
+            Ordered (a, d), (b, e), (c, f).
+        """
         a, b, c, d, e, f, _, _, _ = self
         return (a, d), (b, e), (c, f)
 
     def almost_equals(self, other, precision: float = EPSILON) -> bool:
         """Compare transforms for approximate equality.
 
-        :param other: Transform being compared.
-        :type other: Affine
-        :return: True if absolute difference between each element
-            of each respective transform matrix < ``self.precision``.
+        Parameters
+        ----------
+        other : Affine
+            Transform being compared.
+        precision : float, default EPSILON
+            Precision to use to evaluate equality.
+
+        Returns
+        -------
+        bool
+            True if absolute difference between each element
+            of each respective transform matrix < ``precision``.
         """
         for i in (0, 1, 2, 3, 4, 5):
             if abs(self[i] - other[i]) >= precision:
                 return False
         return True
 
     def __gt__(self, other) -> bool:
+        """Return NotImplemented."""
```

Comment:
I generally don't provide docstrings for dunder methods since their semantics is defined by the language.

```suggestion
```

Reply:
This was done for [undocumented-magic-method (D105)](https://docs.astral.sh/ruff/rules/undocumented-magic-method/#undocumented-magic-method-d105), which is removed from the "ignore" in this PR. I can remove these additions and re-enable the ignore rule?

Reply:
Yes, please.